src/averageAges.js

```diff
@@ -15,13 +17,24 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const male = !century
+    ? personGender('m', people)
+    : personGender('m', people).filter(man =>
+      (century === Math.ceil(man.died / 100)));
```

Comment:
better create another const with result of `personGender`, don't repeat yourself